src/client/pages/ConsentsNewsletters.stories.tsx

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable functional/immutable-data */
+import React from 'react';
+import { Meta } from '@storybook/react';
+
+import { ConsentsNewsletters } from './ConsentsNewsletters';
+
+export default {
+  title: 'Pages/ConsentsNewsletters',
+  component: ConsentsNewsletters,
+} as Meta;
+
+export const NoNewsletters = () => (
+  <ConsentsNewsletters newsletters={[]} isUserInTest={false} />
+);
+NoNewsletters.story = {
+  name: 'with no newsletters',
+};
+
+export const NoNewslettersInTest = () => (
+  <ConsentsNewsletters newsletters={[]} isUserInTest={true} />
+);
+NoNewslettersInTest.story = {
+  name: 'with no newsletters, in test',
+};
+
+export const SingleNewsletter = () => (
+  <ConsentsNewsletters
+    newsletters={[
+      {
+        id: '0',
+        nameId: 'n0',
+        description: 'Newsletter description',
+        name: 'Newsletter Name',
```

Comment:
for one of these stories, it might be worth using an id of a newsletter that has an image, just so we can see what it's like with one e.g. using one of the id's from https://github.com/guardian/gateway/blob/e3b3f2d6d66efe04d03195ac30030a3b1edaa7bb/src/shared/model/Newsletter.ts#L24